scripts/rollup/packaging.js

```diff
@@ -165,15 +166,48 @@ function copyNodePackageTemplate(packageName) {
     // We already created this package (e.g. due to another entry point).
     return Promise.resolve();
   }
-  // TODO: verify that all copied files are either in the "files"
-  // whitelist or implicitly published by npm.
-  return asyncCopyTo(npmFrom, to).then(() =>
-    Promise.all([
+  fs.mkdirSync(to);
+  const packageJson = resolve(`${from}/package.json`);
+  const whitelistedFiles = fs.existsSync(packageJson)
+    ? require(packageJson).files
+    : null;
```

Comment:
Maybe just fall back to `[]`? Then you don't have to check for existence later.

Reply:
Agree. Also need to check `require(packageJson).files` is not `undefined`, as **files** entry is missing in some package.json(e.g. react-call-return).
Updated to 
`const whitelistedFiles = (fs.existsSync(packageJson) &&
    require(packageJson).files) || [];` 
And deleted the existance check below.
~~`if (whitelistedFiles && whitelistedFiles.length > 0) {}`~~
`if (whitelistedFiles.length > 0) {}`